tests/pytests/test_ccresponse.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
```

Comment:
be sure to add this file to the list of test_*.py files in <top-level>/psi4/CMakeLists.txt

Reply:
done, thanks for the reminder!